src/TagField.php

```diff
@@ -200,28 +202,27 @@ public function setSourceList($sourceList)
      */
     public function Field($properties = array())
     {
-        Requirements::css('silverstripe/tagfield:client/dist/styles/bundle.css');
-        Requirements::javascript('silverstripe/tagfield:client/dist/js/bundle.js');
-
-        $schema = [
-            'name' => $this->getName() . '[]',
-            'lazyLoad' => $this->getShouldLazyLoad(),
-            'creatable' => $this->getCanCreate(),
-            'multi' => $this->getIsMultiple(),
-            'value' => $this->Value(),
-            'disabled' => $this->isDisabled() || $this->isReadonly(),
-        ];
-        if (!$this->getShouldLazyLoad()) {
-            $schema['options'] = array_values($this->getOptions()->toNestedArray());
+        Requirements::css('silverstripe/tagfield:css/select2.min.css');
+        Requirements::css('silverstripe/tagfield:css/TagField.css');
+
+        Requirements::javascript('silverstripe/tagfield:js/select2.js');
+        Requirements::javascript('silverstripe/tagfield:js/TagField.js');
```

Comment:
This looks like it's undoing the use of the bundles, maybe going back to what we had in SS3 - is this deliberate?